docs/guides/setup-channel.mdx

```diff
@@ -238,17 +225,45 @@ Alternatively, you can also access quantum processors with REST APIs, enabling y
     * Credentials are saved to `$HOME/.qiskit/qiskit-ibm.json`.  Do not manually edit this file.
     * If you don't save your credentials, you must specify them every time you start a new session.
 
+
     <Admonition type="caution">
-        Account credentials are saved in plain text, so only do so if you are using a trusted device.
+      Account credentials are saved in plain text. To more securely manage your token, it is recommended to use an environment variable; see the [Securely manage your token](#securely-manage) section for more information.
     </Admonition>
 
- 1. Test your setup.  Ensure that you can connect to the service:
+6. Test your setup.  Ensure that you can connect to the service:
 
-   ```python
-    from qiskit_ibm_runtime import QiskitRuntimeService
+```python
+  from qiskit_ibm_runtime import QiskitRuntimeService
 
-    service = QiskitRuntimeService()
-  ```
+  service = QiskitRuntimeService()
+```
+
+<span id="securely-manage"></span>
+## Securely manage your token
+
+To store sensitive information like account credentials more securely, it’s a good practice to use environment variables instead of saving them in plain text. Here’s how to modify your code to load the API token from an environment variable.
+
+1. Set the environment variable.
+
+Set the `IQP_API_TOKEN` environment variable in your system. You can do this by adding the following line to your shell profile (for example, `.bashrc`, `.zshrc`) or by setting it directly in your terminal:
+
+```bash
+export IQP_API_TOKEN="<YOUR_IQP_API_TOKEN>"
+```
+
+2. Modify the code to retrieve the token from the environment variable.
+
+Update your code to retrieve the token from the environment variable instead of saving it directly:
+
+```python
+import os
+from qiskit_ibm_catalog import QiskitFunctionsCatalog
```

Comment:
This example is only relevant for Functions, which not everyone uses. Should this code sample be for Runtime? And then Functions is a subsection?

Reply:
@pandasa123 we could use a Runtime code sample for the setup-channel page, and then use the one you provided for the Functions page (adding a separate section there). Sound ok?

Reply:
But on https://docs.quantum.ibm.com/guides/install-qiskit we don't say to install qiskit-ibm-catalog, and there is no reason to install it unless you're using Functions.

Reply:
@pandasa123 ⬆️

Reply:
> Actually, just spoke to Sanket in-person,
Just casually making us jealous...